documentation/user_docs/docs/manual/Symmetrising_etc.rst

```diff
@@ -124,76 +279,104 @@ Rebin the sqw object ``win`` with the boundaries (and projection axes) of the te
+   sym = [SymopReflection([0,0,1],[0,1,0])
+          SymopReflection([1,0,0],[0,0,1])];
+   fold2 = symmetrise_sqw(hkplane,sym);
 
    %Check the result
    plot(smooth(d2d(fold2)));
 
    %Fold this back again (reverse order of vectors in first fold)
-   fold2a=symmetrise_sqw(fold2,[0,1,0],[0,0,1],[0,0,0]);
+   sym = SymopReflection([0,1,0],[0,0,1]);
+   fold2a = symmetrise_sqw(fold2,sym);
    plot(smooth(d2d(fold2a)))
 
    %Combine with what you started with
-   combi1=combine_sqw(fold2,fold2a);
+   combi1 = combine_sqw(fold2,fold2a);
    plot(smooth(d2d(combi1)));
 
    %Fold back again (reverse order of vectors in second fold)
-   fold3a=symmetrise_sqw(combi1,[0,0,1],[1,0,0],[0,0,0]);
+   sym = SymopReflection([0,0,1],[1,0,0]);
+   fold3a = symmetrise_sqw(combi1, sym);
    plot(fold3a)
 
    %Combine and plot
-   combi2=combine_sqw(combi1,fold3a);
+   combi2 = combine_sqw(combi1,fold3a);
    plot(smooth(d2d(combi2)));
 
 
 Correcting for magnetic form factor
 ***********************************
 
-Horace allows basic correction of scattering intensity from simple ions by adjusting it by the magnetic form factor according to formulas provided in International Tables of Crystallography, Vol C. (see, for example `here <https://www.ill.eu/sites/ccsl/ffacts/ffachtml.html>`__)
+Horace allows basic correction of scattering intensity from simple ions by adjusting it by the magnetic form factor
+according to formulas provided in International Tables of Crystallography, Vol C. (see, for example `here
+<https://www.ill.eu/sites/ccsl/ffacts/ffachtml.html>`__)
 
-The class **MagneticIons** contains the tables of fitting parameters, used to calculate changes in scattering intensity due to changes in magnetic form factor and defines the method *correct_mag_ff*, which takes a memory based **sqw** object as input and returns a similar object, with intensities adjusted by the magnetic form factor:
+The class ``MagneticIons`` contains the tables of fitting parameters, used to calculate changes in scattering intensity
+due to changes in magnetic form factor and defines the method ``correct_mag_ff``, which takes an ``sqw`` object as input
+and returns a similar object, with intensities adjusted by the magnetic form factor:
 
 ::
 
    mff = MagneticIons('Fe0');
    w2_fixed = mff.correct_mag_ff(w2);
 
 
-Where 'Fe0' is the name of the ion for which the magnetic form factor is calculated. **This method should be applied only once**.
+Where 'Fe0' is the name of the ion for which the magnetic form factor is calculated. **This method should be applied
+only once**.
+
+The auxiliary ``MagneticIons``'s method ``IonNames`` returns the cell array of ion names, which are currently tabulated in
+Horace and for which scattering can be corrected using the expression above. Additional ``MagneticIons`` methods
+``calc_mag_ff`` and ``apply_mag_ff`` allow one to calculate magnetic form factor on or apply magnetic form factor to the
+dataset provided.
```

Comment:
really unclear why this is included in the symmetrisation. Sounds as if it should be in a separate doc of its own